dora-parser/src/parser.rs

```diff
@@ -86,98 +86,70 @@ impl<'a> Parser<'a> {
     }
 
     fn parse_top_level_element(&mut self) -> Result<Elem, ParseErrorAndPos> {
-        let modifiers = self.parse_annotation_usages()?;
+        let annotation_usages = self.parse_annotation_usages()?;
 
         match self.token.kind {
             TokenKind::Fun => {
-                self.restrict_modifiers(
```

Comment:
When this is gone, is there still error reporting for illegal annotations?

Reply:
I think this is reintroduced in annotations part 3, after the infrastructure is set in place in this part.